pdns/pdnsutil.cc

```diff
@@ -5242,6 +5248,12 @@ static int listViews(vector<string>& cmds, const std::string_view synopsis)
 
   UtilBackend B("default"); //NOLINT(readability-identifier-length)
 
+  if ((B.getCapabilities() & DNSBackend::CAP_VIEWS) == 0) {
+    // Don't complain about the lack of view support in this case, but
+    // don't list anything either.
+    return 0;
+  }
```

Comment:
oh, right. Now that I see this, I think it's fine (as is the per-view lister that now errors)